extension/Userscripts Extension/SafariWebExtensionHandler.swift

```diff
@@ -188,13 +188,22 @@ class SafariWebExtensionHandler: NSObject, NSExtensionRequestHandling {
         }
         else if name == "PAGE_INIT_DATA" {
             #if os(macOS)
-                if let settings = getInitData(), checkDefaultDirectories() {
-                    response.userInfo = [SFExtensionMessageKey: settings]
+                if let initData = getInitData(), checkDefaultDirectories() {
+                    response.userInfo = [SFExtensionMessageKey: initData]
                 } else {
                     response.userInfo = [SFExtensionMessageKey: ["error": "failed to get init data"]]
                 }
             #endif
         }
+        else if name == "PAGE_LEGACY_IMPORT" {
```

Comment:
You mentioned this is the `settings.js` file:

```
// import complete tag, to ensure will only be import once
Object.assign(settings, {"legacy_imported": Date.now()});
```

I just want to verify that the `legacy_import` function only runs one time for a user; is that right? Eventually this legacy import functionality will be removed?